website/content/api-docs/system/leases.mdx

```diff
@@ -243,3 +243,68 @@ $ curl \
     --request POST \
     http://127.0.0.1:8200/v1/sys/leases/tidy
 ```
+
+## Irrevocable Leases Counts
+
+This endpoint returns the total count of irrevocable leases, as well as a count
+per mount point.
+
+This can help determine if particular endpoints are disproportionately
+resulting in irrevocable leases.
+
+This endpoint was added in Vault 1.8.
+
+### Parameters
+
+- `type` (string: <required>) - Specifies the type of lease. Set to "irrevocable"
+- `include_child_namespaces` (bool: false) - Specifies if irrevocable leases in
+   child namespaces should be included in the result
+
+| Method | Path               |
+| :----- | :----------------- |
+| `GET`  | `/sys/leases/count`|
+
+### Sample Request
+
+```shell-session
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request GET \
+    http://127.0.0.1:8200/v1/sys/leases/count \
+    -d type=irrevocable
+```
+
+## Irrevocable Leases List
```

Comment:
given the `type` parameter, should the documentation around this API be a little more general?

Reply:
e5dafb4cd8